client.go

```diff
@@ -405,7 +410,7 @@ func unixDial(networ, addr string) (net.Conn, error) {
 		if err != nil {
 			return nil, fmt.Errorf(gettext.Gettext("cannot resolve unix socket address: %v"), err)
 		}
-	} else {
+	} else { // TODO - I think this is dead code
```

Comment:
On Sat, Mar 21, 2015 at 06:19:39PM -0700, St�phane Graber wrote:
> > @@ -405,7 +410,7 @@ func unixDial(networ, addr string) (net.Conn, error) {
> >  		if err != nil {
> >  			return nil, fmt.Errorf(gettext.Gettext("cannot resolve unix socket address: %v"), err)
> >  		}
> > -	} else {
> > +	} else { // TODO - I think this is dead code
> 
> Really? Isn't that code path required for any socket which isn't in /var/lib/lxd ?

No.

Because any such socket will be using the uDial function.

> The other codepath has an hardcoded socket path, so if the else is never used, we've got another problem :)
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/lxc/lxd/pull/417/files#r26898792